src/sas/qtgui/MainWindow/DataExplorer.py

```diff
@@ -1326,16 +1335,14 @@ def chooseFiles(self):
         caption = 'Choose files'
         directory = self.default_load_location
         filter = wlist
-        options = QtWidgets.QFileDialog.DontUseNativeDialog | QtWidgets.QFileDialog.DontUseCustomDirectoryIcons
-        paths = QtWidgets.QFileDialog.getOpenFileNames(parent, caption, directory, filter, options=options)[0]
+        paths = QtWidgets.QFileDialog.getOpenFileNames(parent, caption, directory, filter)[0]
 
```

Comment:
The reason we used `DontUseCustomDirectoryIcons` was the occasional horrendous wait times when navigating to a directory with a number of subdirectories/files. This was a crippling bug on some windows machines.

This _might_ have been fixed in current version of PySide6 but please test.

Reply:
I've been having issues with the dialog since we moved to PySide6 where it won't display any files or folders except the current folder. Removing these options has fixed that on Windows, but I'll test on Mac.

As a side note - I realize these options are in a few other places and should be removed from all places if we go this route.